modules/connectivity/variables.tf

```diff
@@ -119,6 +119,7 @@ variable "settings" {
             enabled = optional(bool, false)
             config = optional(object({
               address_prefix                = optional(string, "")
+              address_management_prefix     = optional(string, "")
```

Comment:
Interested to understand why the management subnet was added? Apologies for my lack of knowledge in this area. I see it was not needed for the other SKU, so wondering if this is just an improvement to the module of if it is needed for Basic?

Reply:
Yes I know what you mean it was a headscratcher for me too since the other SKUs do not need this subnet except for an azure firewall in forced tunnelling mode. However as per the deployment guidance the subnet is mandatory for the basic SKU. 
"_As the Azure Firewall Basic has limited traffic compared to the Azure Firewall Standard or Premium SKU, it requires the AzureFirewallManagementSubnet to separate customer traffic from Microsoft management traffic to ensure no disruptions on it. This management traffic is needed for updates and health metrics communication that occurs automatically to and from Microsoft only. No other connections are allowed on this IP_"

https://learn.microsoft.com/en-us/azure/firewall/deploy-firewall-basic-portal-policy